src/components/Autocomplete/components/ComboBox/ComboBox.tsx

```diff
@@ -7,7 +7,7 @@ import {PreferredPosition} from '../../../PositionedOverlay';
 import {ActionListItemDescriptor, Key} from '../../../../types';
 import {contextTypes} from '../types';
 import KeypressListener from '../../../KeypressListener';
-import {TextField} from './components';
```

Comment:
With the change in export, the path to the TextField from the [ComboBox test TSX file needs to be updated ](https://circleci.com/gh/Shopify/polaris-react/22335?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).